.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_PATH: "vendor/bundle"
+BUNDLE_WITHOUT: "screenshots"
```

Comment:
+1 for tracking this 👍 

Worth reminding that the issue which made us remove it still isn't fixed: See https://github.com/rubygems/rubygems/issues/3708 and https://github.com/rubygems/rubygems/pull/4152.

That shouldn't be a problem in CI, though, because it doesn't need to generate screenshots at this point in time. Even if it did, I think leaving the repo dirty shouldn't be a problem there.

As for local devs, I think there's few enough people that work on the screenshots automation that we can live with the workaround knowledge (discard the changes to this file produced by `BUNDLE_WITH=screenshots bundle install`) tracked orally.

Reply:
Yep – I'd love to make this not even necessary – IIRC that would just require aligning with the same `imagemagick` gem that `fastlane` already uses internally.